app/Http/Controllers/AmaController.php

```diff
@@ -1,32 +1,20 @@
 <?php
 namespace App\Http\Controllers;
-ini_set('display_errors', 'On');
 
 use Illuminate\Routing\Controller as BaseController;
+use App\Ama as Ama;
+use App\ViewModel as ViewModel;
 
 class AmaController extends BaseController
 {
     public function index()
     {
-        $ama1 = new Ama();
-        $ama1->title = 'Ama 1';
-        $ama1->text = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
-        $ama2 = new Ama();
-        $ama2->title = 'Ama 2';
-        $ama2->text = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
-        $ama3 = new Ama();
-        $ama3->title = 'Ama 3';
-        $ama3->text = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
+        $vm = new ViewModel();
```

Comment:
You don't need a `ViewModel` class. Just do `$amas = Ama::all();` and then send the `$amas` to the view as you did the first time. `return view('index', ['amas' => $amas]);` 
If you also want to send the image, you can use a separate variable `$imageUrl = 'img/home-bg.jpg';` and then add it to the array you pass to the view: `return view('index', ['amas' => $amas, 'imageUrl' => $imageUrl]);`